<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_Tanks">
       <attribute name="title">
        <string>Tanks</string>
       </attribute>
       <widget class="QTableWidget" name="tableWidget_tanks">
        <property name="geometry">
         <rect>
          <x>11</x>
          <y>11</y>
          <width>661</width>
          <height>192</height>
         </rect>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox_add_payment">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>280</y>
          <width>221</width>
          <height>211</height>
         </rect>
        </property>
        <property name="title">
         <string/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_add_payment_customer">
             <property name="text">
              <string>Customer</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_add_payment_customer"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_add_payment_tank">
             <property name="text">
              <string>Tank</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_add_payment_tank"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_add_payment_cantity">
             <property name="text">
              <string>Cantity</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="doubleSpinBox_add_payment_cantity"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label_add_payment_total">
             <property name="text">
              <string>Total</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_add_payment_total">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_add_payment">
           <property name="text">
            <string>Add a payment</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QGroupBox" name="groupBox_refill_tank">
        <property name="geometry">
         <rect>
          <x>470</x>
          <y>300</y>
          <width>231</width>
          <height>181</height>
         </rect>
        </property>
        <property name="title">
         <string/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_refill_tank_tank">
             <property name="text">
              <string>Tank</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_refill_tank_tank"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_refill_tank_cantity">
             <property name="text">
              <string>Cantity</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="doubleSpinBox_refill_tank_cantity"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_refill_tank">
           <property name="text">
            <string>Refill tank</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="label_price">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>220</y>
          <width>41</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Price:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_price">
        <property name="geometry">
         <rect>
          <x>75</x>
          <y>220</y>
          <width>40</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>1.00</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_price_ron">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>220</y>
          <width>31</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>RON</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_save_price">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>220</y>
          <width>93</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Save price</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox_enable_price">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>220</y>
          <width>131</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>enable change price</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_Customers">
       <attribute name="title">
        <string>Customers</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_Employees">
       <attribute name="title">
        <string>Employees</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
